app/services/recent-events.ts

```diff
@@ -123,14 +185,19 @@ export class RecentEventsService extends StatefulService<IRecentEventsState> {
 
   private async formEventsArray() {
     const events = await this.fetchRecentEvents();
+    // const readReciepts = await this.fetchReadReceipts();
     let eventArray: IRecentEvent[] = [];
     if (!events.data) return;
     Object.keys(events.data).forEach(key => {
+      const fortifiedEvents: IRecentEvent[] = events.data[key].map(event => {
```

Comment:
I think the type annotation isn't needed here or on `culledEvents` below.  It was left over from some older code but I think we can try removing and see if the type inference figures this out.

Reply:
Yup wasn't needed